tests/lifecycle/tests/lifecycle_shutdown.go

```diff
@@ -62,11 +70,181 @@ var _ = Describe("lifecycle lifecycle-container-shutdown", func() {
 		Expect(err).To(HaveOccurred())
 
 		By("Verify test case status in Junit and Claim reports")
-		err = nethelper.ValidateIfReportsAreValid(
+		err = globalhelper.ValidateIfReportsAreValid(
 			lifeparameters.ShutdownDefaultName,
 			globalparameters.TestCaseFailed)
 		Expect(err).ToNot(HaveOccurred())
 
 	})
 
+	// 47382
+	It("One deployment, several pods, several containers that "+
+		"have preStop field configured", func() {
+
+		By("Define deployment with preStop field configured")
+		replicaDefinedDeployment := globalhelper.AppendContainersToDeployment(
```

Comment:
Basically these lines are duplicated across all tests 

```
globalhelper.AppendContainersToDeployment(
			deployment.RedefineWithReplicaNumber(
				deployment.DefineDeployment(
					"lifecycleput",
					lifeparameters.LifecycleNamespace,
					globalhelper.Configuration.General.TnfImage,
					lifeparameters.TestDeploymentLabels), 3),
			2,
			globalhelper.Configuration.General.TnfImage)

		preStopDeploymentStruct := deployment.RedefineAllContainersWithPreStopSpec(
			replicaDefinedDeployment, lifeparameters.PreStopCommand)

```
Can you move them in to a new helper function.
Something like: 
func DefineLifecycleDeploymentPreStop(preStop bool) {
...
}